docs/README.md

```diff
@@ -60,3 +60,4 @@ The following are a few details about other files Docusaurus uses that you may w
 - `../babel.config.js`: Babel config file used when building
 - `../package.json`: dependencies and scripts
 - `../yarn.lock`: dependency lock file that ensures reproducibility
+- `sitemap.xml`: After any changes to sidebars.js use https://www.xml-sitemaps.com/ with the docs url https://docs.greatexpectations.io/docs/ and then download the resulting sitemap.xml to update the existing
```

Comment:
update the existing ....? Also, should an update to sitemap.xml be included in this this change since sidebar was modified?

Reply:
Good question - I clarified this comment, let me know if the new version is clearer. Also I realized that I can generate the sitemap with the temp netlify url and manually edit to the appropriate url, doing that now.